### PR TITLE
MNT #59: Change motor names for mono

### PR DESCRIPTION
**Pull request by qzhang234**
@prjemian @sureshnaps 
The names for motors under mono have been changed per Pete's request

### CODE REVIEW

File: profile_bluesky/startup/10-devices.py [hunk 1]
```diff
@@ -12,10 +12,10 @@ class CompoundRefractiveLensDevice(Device):
 
 
 class MonochromatorDevice(Device):
-    monoE = Component(EpicsMotor, '8idimono:sm2', labels=["motor", "mono", "optics"])
-    monoth = Component(EpicsMotor, '8idimono:sm1', labels=["motor", "mono", "optics"])
+    Energy = Component(EpicsMotor, '8idimono:sm2', labels=["motor", "mono", "optics"])
```
Comment (prjemian) [inline profile_bluesky/startup/10-devices.py]: `energy`, not `Energy`

Comment (qzhang234) [reply profile_bluesky/startup/10-devices.py]: @prjemian It has been fixed. Shall I generate a new Merge Pull Request?

**Comment by prjemian** [comment]
@qzhang234 : You can merge this now, then delete the branch

**Comment by qzhang234** [comment]
fixes #59

**Comment by qzhang234** [comment]
@prjemian Done. Please verify

**Pull request merged**